### PR TITLE
Update for CsWinRT 2 - net6.0 and ILTrimming

### PR DESCRIPTION
**Pull request by j0shuams**

### CODE REVIEW

File: build/NuSpecs/Microsoft.WindowsAppSDK.Foundation.props [hunk 1]
```diff
@@ -1,4 +1,7 @@
-﻿<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
```
Comment (manodasanW) [inline build/NuSpecs/Microsoft.WindowsAppSDK.Foundation.props:1]: Looks like random characters got added at start?

File: dev/MRTCore/mrt/Microsoft.Windows.ApplicationModel.Resources/projection/Microsoft.Windows.ApplicationModel.Resources.Projection.csproj [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   
   <PropertyGroup>
-    <TargetFramework>net5.0-windows10.0.17763.0</TargetFramework>
+    <TargetFramework>net6.0-windows10.0.17763.0</TargetFramework>
```
Comment (manodasanW) [inline dev/MRTCore/mrt/Microsoft.Windows.ApplicationModel.Resources/projection/Microsoft.Windows.ApplicationModel.Resources.Projection.csproj:4]: Enable trimming?

**Comment by manodasanW** [comment]
These might also need to be updated:

https://github.com/microsoft/WindowsAppSDK/blob/96837507aa4016d4a32377cf38a29f8155dd4a2d/dev/MRTCore/global.json

**Comment by j0shuams** [comment]
> These might also need to be updated:
> 
> https://github.com/microsoft/WindowsAppSDK/blob/96837507aa4016d4a32377cf38a29f8155dd4a2d/dev/MRTCore/global.json

Maybe? There's no dotnet version in this file on main

**Review by manodasanW — APPROVED** [review]

**Pull request merged**